### DISCUSSION TITLE
About the bug of tool sql-schema-to-swift-struct

### DISCUSSION THREAD

**Discussion by Rigadin** [he3-app/he3-feedback#262] [Bug Report]
### Version
sql-schema-to-swift-struct 1.0.5

### Environment
Mac v1.3.23

### What is expected?
Converting a SQL table into a swift structure

### What is actually happening?

The swift file is not generated. I searched in vain for a button, menu or shortcut to perform the conversion.
I must have missed something :-)

**Comment by lyzhang1999** [comment]
Please enter the content and paste it

**Comment by Rigadin** [comment]
Hi,

"A picture is worth a thousand words"

￼
